### ISSUE TITLE
[FEEDBACK] Example needs more than one domain

### ISSUE THREAD

**Issue by jeserkin** [phalt/django-api-domains#34]
I did find given styleguide useful, but slightly limited in sense, that normal application would probably/might have request/response objects, that also should be mentioned and example should cover, where they could be located.
Model is not necessary one and the same in that sense.

Also since domain is being covered then in example of the shop you might encounter domains such as Product, Order, Price. It would be nice to see how those domain in example would cross paths.

Thank you.

**Comment by phalt** [comment]
Yeah, you're right. More complex examples are needed. I don't actively work on this and I loosely follow the pattern in my current job, but I can't easily lift and shift examples out of that without potentially violating Intellectual Property.

I'll have a think and see what I can do.

**Comment by lexer21** [comment]
Any update on this? Would really nice to get some more examples with DRF
